pkg/kev/skaffold.go

```diff
@@ -81,11 +81,8 @@ var (
 // NewSkaffoldManifest returns a new SkaffoldManifest struct.
 func NewSkaffoldManifest(envs []string, project *ComposeProject) (*SkaffoldManifest, error) {
```

Comment:
`NewSkaffoldManifest` only returns a `nil` error. Maybe remove the returned `error` arg.